src/UlozenkaLib/APIv3/Formatter/JsonFormatter.php

```diff
@@ -410,6 +443,49 @@ private function proccessStatusHistoryResponseData($dataObject)
     /**
      *
      * @param stdClass $dataObject
+     *
+     * @return \UlozenkaLib\APIv3\Model\Tracking
+     */
+    private function proccessTrackingResponseData($dataObject)
```

Comment:
Zde se omlouvám, že to není nikde popsané, ale rád bych se držel toho že metoda getData() vrací stejně jako API pole. Zde bych tedy vrátil pole objektů typu Tracking obdobně jako u jiných zdrojů (processXXXResponseData). Protože se ale jedná o zdroj, který bude mít (v aktuální implementaci) vždy v datech jen jeden objekt, je možné do třídy GetTrackingResponse přidat další metodu, která vr8t9 tento objekt }ekvivalent k getLabelsString() ze třídy GetLabelsResponse.